CSharp/Library/Microsoft.Bot.Connector.Shared/ConnectorAPI/Models/AudioCard.cs

```diff
@@ -73,13 +73,13 @@ public AudioCard(string title = default(string), string subtitle = default(strin
         /// Array of media Url objects
         /// </summary>
         [JsonProperty(PropertyName = "media")]
-        public IList<MediaUrl> Media { get; set; }
+        public IList<MediaUrl> Media { get; set; } = new MediaUrl[0];
```

Comment:
>MediaUrl [](start = 57, length = 8)

while I generally like this change, it seems like it will only apply on outgoing activities, because the json deserializer won't do the same for incoming activities?